frontend/src/features/mapSideBar/SidebarContents/LandForm.tsx

```diff
@@ -0,0 +1,300 @@
+
+  values.data.evaluations = _.filter(allFinancials, financial =>
+    Object.keys(EvaluationKeys).includes(financial.key),
+  );
+  values.data.fiscals = _.filter(allFinancials, financial =>
+    Object.keys(FiscalKeys).includes(financial.key),
+  );
+  values.data.landArea = +values.data.landArea;
+  values.data.financials = [];
+  return values.data;
+};
+
+interface ILandSubformProps {
+  handlePidChange: (pid: string) => void;
+  handlePinChange: (pin: string) => void;
+  setMovingPinNameSpace: (nameSpace: string) => void;
+  handleGeocoderChanges: any;
+}
+
+const Form: React.FC<ILandSubformProps> = ({
+  handleGeocoderChanges,
+  setMovingPinNameSpace,
+  handlePidChange,
+  handlePinChange,
+}) => {
+  // access the stepper to later split the form into segments
+  const stepper = useFormStepper();
+  const formikProps = useFormikContext<IParcel>();
+
+  // lookup codes that will be used by subforms
+  const { getOptionsByType } = useCodeLookups();
+  const agencies = getOptionsByType(API.AGENCY_CODE_SET_NAME);
+  const classifications = getOptionsByType(API.PROPERTY_CLASSIFICATION_CODE_SET_NAME);
+
+  const render = (): React.ReactNode => {
+    switch (stepper.current) {
+      case 0:
+        return (
+          <div className="parcel-identification">
+            <ParcelIdentificationForm
+              agencies={agencies}
+              classifications={classifications}
+              handleGeocoderChanges={handleGeocoderChanges}
+              setMovingPinNameSpace={setMovingPinNameSpace}
+              handlePidChange={handlePidChange}
+              handlePinChange={handlePinChange}
+            />
+          </div>
+        );
+      case 1:
+        return (
+          <div className="parcel-usage">
+            <LandUsageForm classifications={classifications} nameSpace="data" {...formikProps} />
+          </div>
+        );
+      case 2:
+        return <LandValuationForm />;
+      case 3:
+        return <LandReviewPage classifications={classifications} agencies={agencies} />;
+    }
+  };
+  return (
+    <FormContentWrapper>
+      <FormContent>{render()}</FormContent>
+      <FormFooter>
+        <InventoryPolicy />
+        <FillRemainingSpace />
+        {stepper.current !== 3 && (
+          <Button
+            type="button"
+            onClick={() => {
+              stepper.gotoNext();
+            }}
+            size="sm"
+          >
+            Continue
+          </Button>
+        )}
+        {formikProps.dirty && formikProps.isValid && stepper.current === 3 && (
+          <Button size="sm" type="submit">
+            Submit Raw Land
+          </Button>
+        )}
+      </FormFooter>
+    </FormContentWrapper>
+  );
+};
+
+interface ILandForm {
+  formikRef?: any;
+  handleGeocoderChanges?: any;
+  setMovingPinNameSpace: (nameSpace: string) => void;
+  handlePidChange: (pid: string) => void;
+  handlePinChange: (pin: string) => void;
+}
+
+const LandForm: React.FC<ILandForm> = (props: ILandForm) => {
```

Comment:
Should separate these types of multi-components into multiple files but organize them so they are all still in the same folder if they are part of a larger component.